### PR TITLE
FLOE-289: Add security screencast and update links to prefs editors screencast.

### PR DESCRIPTION
**Pull request by acheetham**
@jobara, @michelled has reviewed the text for this pull request. Would be able to test that it all works?

### CODE REVIEW

File: security.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    </div>
+                    
+                    <div class="floe-column">
+                        <p>
+                            205 Richmond St. W., Second Floor<br/>
+                            Toronto, Ontario, Canada<br/>
+                            M5V 1V3<br/>
+                            Telephone: (416) 977-6000 #3968<br/>
+                            Fax: (416) 977-9844
+                        </p>
```
Comment (jobara) [inline security.html]: Could we put the address in an `<address>` element?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

File: security.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                    with any GPII-enabled services.</p>
+                <p>For more information and to see an example of the security architecture in action,
+                    please see the video below.</p>
+                <video controls width="640px" height="360px" tabindex="0">
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.mp4" type="video/mp4"></source>
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.webm" type="video/webm"></source>
+                </video>
+            </div>
+        
+            <div class="floe-footer">
```
Comment (jobara) [inline security.html]: can we make this a `<footer>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer

File: security.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+            <a href="#navigation">Skip to Navigation</a>
+        </div>
+        
+        <div class="floe">
+            <div class="floe-header">
```
Comment (jobara) [inline security.html]: Should we make this a `<header>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

File: security.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+            <a href="#navigation">Skip to Navigation</a>
```
Comment (jobara) [inline security.html]: Is there a navigation section to skip to? Maybe we should remove this if there isn't.

**Comment by jobara** [comment]
@acheetham I also noticed that the index, prefsEditors, and security pages all have the same page title. Is that expected?

**Comment by acheetham** [comment]
Hm. Good question, @jobara. I suspect we didn't think about it, since the site has so few pages. I'll come up with better names.

**Comment by acheetham** [comment]
@jobara, this is ready for another review

File: security.html (hunk 1 above)
Comment (jobara) [inline security.html]: One more place for an `<address>` block

**Comment by jobara** [comment]
merged at 0dd79bab7973baa64c021a027ad811e213704a42

**Pull request merged**